src/OpenLoco/src/Ui/Widget.h

```diff
@@ -22,6 +22,13 @@ namespace OpenLoco::Ui
     struct Window;
     enum class WindowColour : uint8_t;
 
+    enum class ContentAlign
```

Comment:
won't this ultimately become something we use from the graphics engine. Maybe we should move it into a different header already

Reply:
No, this drives how widgets in the future will have their content aligned.
![image](https://github.com/user-attachments/assets/bb3524cc-4e47-4f3b-a913-eb4a484f0079)
Like in this image its three labels, first is align left, second is align center, third is align right. Since one of the goals is to have widgets having children this will specify where the content will be, if that is a label or a panel or something else.